src/app/pages/pages-routing.module.ts

```diff
@@ -35,6 +36,9 @@ const routes: Routes = [{
     path: '',
     redirectTo: 'dashboard',
     pathMatch: 'full',
+  }, {
+    path: '**',
+    component: PageNotFoundComponent,
```

Comment:
Also, let's name it `NotFoundComponent` as the other pages don't have a `Page` prefix.